tests/test_endpoints.py

```diff
@@ -108,3 +122,8 @@ def test_embedding_endpoint_ollama():
 def test_llm_endpoint_anthropic():
     filename = os.path.join(llm_config_dir, "anthropic.json")
     run_llm_endpoint(filename)
+
+
+def test_llm_endpoint_groq():
+    filename = os.path.join(llm_config_dir, "groq.json")
```

Comment:
We're missing `groq.json` in this PR I believe? At least I don't see it here or in `main`: https://github.com/cpacker/MemGPT/tree/main/configs/llm_model_configs

Reply:
Interesting...didn't get pushed. Not sure why.

Reply:
It's in the gitignore actually